content/posts/newsletter-february-2024.md

```diff
@@ -0,0 +1,32 @@
+title: Read the Docs Newsletter - February 2024
+date: Jan 1, 2024
+description: Company updates and new features from the last month, current focus, and upcoming features.
+category: Newsletter
+tags: newsletter, community, build-in-public
+authors: Eric Holscher
+status: published
+
+## News and updates
+
+* We moved our new blog posts to be published on our website, so going forward you can find it [directly on our website](https://about.readthedocs.org/blog/).
+* We shipped an update that allows [Sphinx builds to share environments](https://github.com/readthedocs/readthedocs.org/pull/11073). This should make builds of multiple formats faster.
+* We shipped a few different security fixes that were found by Santos Gallegos, a member of our team. You can read more about them in our [security advisories](https://github.com/readthedocs/readthedocs.org/security/advisories)
+* We shipped an [updated settings page for Addons](https://github.com/readthedocs/readthedocs.org/pull/11031) in our new beta dashboard. This allows enabling and disabling each addon.
+
+You can always see the latest changes to our platforms in our [changelog 📃](https://docs.readthedocs.io/page/changelog.html).
+
+## Upcoming changes
+
+- We continue to work on improving Addons. Our goal is to default all projects to using Addons in Q2 of 2024. We are working on improving the user experience, and adding more Addons to make the transition easier.
+
+Want to follow along with our development progress? View our [development roadmap 📍️](https://github.com/orgs/readthedocs/projects/156/views/1)
+
+## Possible issues
```

Comment:
Maybe mention that users should update their readthedocs-sphinx-search extension? We also emailed users about it https://github.com/readthedocs/readthedocs-sphinx-search/security/advisories/GHSA-xgfm-fjx6-62mj.